README.md

```diff
@@ -20,27 +28,42 @@ You may use the contents of this repo in parts or in whole according to the 0BSD
 > OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH
 > THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-> **This repository does not accept Pull Requests at the moment.**
-
-## Components
-
-You can find a Docker Compose setup here with the following services:
-
-* Daml Platform:
-  * All-in-one Canton node (domain topology manager, mediator, sequencer and participant)
-  * HTTP JSON API service
-  * PostgreSQL database (used by the Canton node and the HTTP JSON API service)
-* Monitoring:
-  * Prometheus `2.x`
-  * Grafana `9.x`
-  * Node Exporter
-  * Loki + Promtail `2.x`
-
-Prometheus and Loki are [preconfigured as datasources for Grafana](./grafana/datasources.yml). You can add other
-services/exporters in the [Docker compose file](./docker-compose.yml) and scrape them changing the
-[Prometheus configuration](./prometheus/prometheus.yml).
-
-Use the `Makefile` — run `make help` for available commands!
+## TL;DR Quick Start
+
+To quickly get up and running, make sure you have the [prerequisites](#Prerequisites) installed
+and then:
+1. Ensure you have enough resources allocated to running the example
+   containers; if a memory resource limit is reached then a container will be
+   randomly killed.  In particular, the ```daml_observability_canton```
+   container can reach a steady state memory usage of 4 GB.
+2. Deploy the containers locally, which include several Daml Enterprise
+components, Prometheus, Grafana, and related containers.  This is done withe
+command:
+```docker compose -p obs up```.
+3. Start the request generation.  There are three scripts that generate the requests
+   and running them in separate shell consoles is best.  The scripts are:
+* ```scripts/generate-grpc-load.sh <number loops>``` this runs the ledger API
```

Comment:
Suggestion to visually separate the script from its description:
```suggestion
* ```scripts/generate-grpc-load.sh <number loops>```: this runs the ledger API
```